cmd-info.c

```diff
@@ -44,10 +44,9 @@ static char *copy_info_str(char *src)
 	return dst;
 }
 
-static int fill_exe_name(void *arg)
+static int fill_exe_name(void *arg, char *buf)
 {
 	struct fill_handler_arg *fha = arg;
```

Comment:
It'd be better if the struct arg have the buffer.  You could define `struct read_handler_arg` as well.

Reply:
I got it !

Reply:
> You could define struct read_handler_arg as well.

Hum.. @namhyung , I didn't understand your comment:
I just add `char * buf[PATH_MAX]` to both  `struct fill_handler_arg` 
and `struct ftrace_file_handle`. Isn't it what you mean ?